### ISSUE TITLE
Wrong bus for plotting_toolbox.position_on_busbar() in collections.py

### ISSUE THREAD

**Issue by marhofmann** [e2nIEE/pandapower#706]
Th bus for plotting_toolbox.position_on_busbar() in collections.create_line_switch_collection() should be 'sb' not target_bus! Otherwise it's not working...

`
intersection = position_on_busbar(net, sb, busbar_coords=line_coords)
`

**Comment by marhofmann** [comment]
https://github.com/e2nIEE/pandapower/blob/50768df269aa93dd247bb8572709aff2ec934213/pandapower/plotting/collections.py#L970

**Comment by lthurner** [comment]
I don't really understand this code on the first glance, didn't you yourself contribute this part? I guess you should know best then :D

Can you maybe provide a short example of a grid that plots wrong with the current implementation?

**Comment by marhofmann** [comment]
Hi Leon, 

sorry for the late reply. You are right, I contributed this line and the dedicated function by myself! But since the 'position_on_busbar' has been renamed and moved to 'plotting_toolbox.py' I though somthing went wrong. I checked it with my PR and found out that the problem is sitting right in front of my screen :-D
I can't figure out why i didn't notice it earlier since i tested it quite well before pushing... Anyway - I am very sorry about that!
But let us fix it now...

By creating a simple example i noticed, that there is no documentation for 'create_busbar_collection' [in](https://pandapower.readthedocs.io/en/v2.2.2/elements/line.html) 

I created a simple example which should illustrate the problem:

`
import pandapower as pp
import pandapower.plotting as ppp

net = pp.create_empty_network()
bus = pp.create_bus(net, vn_kv=20, geodata=(0, 0))
busbar = pp.create_bus(net, vn_kv=20, geodata=(12, 20), type='b', coords=[[(10, 20), (25, 20)]])
line = pp.create_line(net, from_bus=bus, to_bus=busbar, geodata=[(0, 0), (20, 20)], length_km=2, std_type="NA2XS2Y 1x150 RM/25 12/20 kV")
pp.create_switch(net, bus=busbar, element=line, et='l')

bc = ppp.create_bus_collection(net, buses=[bus], size=1)
bbc = ppp.create_busbar_collection(net, linewidth=1)
lc = ppp.create_line_collection(net, color='k')
sc = ppp.create_line_switch_collection(net)

ppp.draw_collections(collections=[bc, bbc, lc, sc])
`
It produces this output:

![simple_ex](https://user-images.githubusercontent.com/47354890/77568940-b5744800-6ec9-11ea-8586-b16450ee67ce.png)


Changeing it like mentioned before and it looks like this:

![simple_ex2](https://user-images.githubusercontent.com/47354890/77569098-f2d8d580-6ec9-11ea-9413-e1317d77dc29.png)

**Comment by marhofmann** [comment]
Hi Leon, 

sorry for the late reply. You are right, I contributed this line and the dedicated function by myself! But since the 'position_on_busbar' has been renamed and moved to 'plotting_toolbox.py' I though somthing went wrong. I checked it with my PR and found out that the problem is sitting right in front of my screen :-D
I can't figure out why i didn't notice it earlier since i tested it quite well before pushing... Anyway - I am very sorry about that!
But let us fix it now...

By creating a simple example i noticed, that there is no documentation for 'create_busbar_collection' [in](https://pandapower.readthedocs.io/en/v2.2.2/elements/line.html) 

I created a simple example which should illustrate the problem:
import pandapower as pp
import pandapower.plotting as ppp

```
import pandapower as pp
import pandapower.plotting as ppp

net = pp.create_empty_network()
bus = pp.create_bus(net, vn_kv=20, geodata=(0, 0))
busbar = pp.create_bus(net, vn_kv=20, geodata=(12, 20), type='b', coords=[[(10, 20), (25, 20)]])
line = pp.create_line(net, from_bus=bus, to_bus=busbar, geodata=[(0, 0), (12, 20)], length_km=2, std_type="NA2XS2Y 1x150 RM/25 12/20 kV")
pp.create_switch(net, bus=busbar, element=line, et='l')

bc = ppp.create_bus_collection(net, buses=[bus], size=1)
bbc = ppp.create_busbar_collection(net, linewidth=1)
lc = ppp.create_line_collection(net, color='k' )
sc = ppp.create_line_switch_collection(net, distance_to_bus=2, use_line_geodata=True)

ppp.draw_collections(collections=[bc, bbc, lc, sc])
```
Produces this output:
![simple_ex](https://user-images.githubusercontent.com/47354890/77571255-50baec80-6ecd-11ea-90d2-3bb10f38b49d.png)

But the switch is dedicated to the busbar and not to the node! 
Applying the changes mentioned above produces this output:
![simple_ex2](https://user-images.githubusercontent.com/47354890/77571335-7647f600-6ecd-11ea-8140-985ecc1789ca.png)
